ComprefaceTestApp/Program.cs

```diff
@@ -1,14 +1,17 @@
-﻿using System.Text.Json;
+﻿using System.Reflection;
+using System.Text.Json;
+using ComprefaceTestApp.Configuration;
 using ComprefaceTestApp.DTOs.ExampleSubject.AddExampleSubject;
 using ComprefaceTestApp.DTOs.ExampleSubject.ListAllExampleSubject;
 using ComprefaceTestApp.DTOs.SubjectDTOs.AddSubject;
 using ComprefaceTestApp.DTOs.SubjectDTOs.RenameSubject;
 using ComprefaceTestApp.Services;
 using Flurl.Http;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
 using Shared;
-using Shared.Constants;
 
 namespace ComprefaceTestApp;
 
```

Comment:
Please create new class and interface IComprefaceClient. During startup all DI containers should raise up. Implement IDisposable. Program should looks like var client = new ComprefaceClient(apiKey, host); var result = client.CheckImage(img);

Reply:
Done